x/ibc/03-connection/keeper/keeper_test.go

```diff
@@ -50,20 +51,32 @@ var (
 type KeeperTestSuite struct {
 	suite.Suite
 
+	coordinator *ibctesting.Coordinator
+
+	// testing chains used for convenience and readability
+	chainA *ibctesting.TestChain
+	chainB *ibctesting.TestChain
+
+	// TODO: delete
 	cdc *codec.Codec
 
 	// ChainA testing fields
-	chainA *TestChain
+	oldchainA *TestChain
```

Comment:
these are just place holders so old tests don't fail, but I can still refactor this pkg one file at a time, will be deleted once the last file is refactored